### ISSUE TITLE
scripts in ch4 should be modified

### ISSUE THREAD

**Issue by Watesoyan** [megansquire/masteringDM#2]
`networkx.degree(...)` yields a `networkx.classes.reportviews.DegreeView` object, 
which has no method called `*.values()`
so, I think:

1. line 16 in basicNetworkMetrics.py should be modified to `dict(nx.degree(g))`
2. line 6 in basicNetworkMetrics2.py to `for n in g.nodes():`
   line 27 to `d3 = dict(nx.degree(g2))`
4. line 23 in basicNetworkMetrics3.py to `graphsSorted = sorted(graphs, key=len, reverse=True)`
   line 31 to `graphDegree = dict(nx.degree(graph))`
6. line 11 in basicNetworkMetrics5.py to `graphDegree = dict(nx.degree(g))`
7. line 10 in basicNetworkMetrics6.py to `graphs = sorted(list(nx.connected_component_subgraphs(g)), key=len, reverse=True) `
    line 15 to `graphDegree = dict(nx.degree(ego))`
9. line 18 in basicNetworkMetrics7.py to `graphDegree = dict(nx.degree(graph))`
10. line 17 in basicNetworkMetrics8.py to `graphDegree = dict(nx.degree(ego))`
11. line 19 in basicNetworkMetrics9.py to `graphDegree = dict(nx.degree(ego))`
12. line 24 in bonusWeightedGraph.py to `d = dict(nx.degree(ego))`

**Comment by ghost** [comment]
Yeah.I find it either.Maybe older version is Dict, but now it's DegreeView object.
